src/app/resources/iis_scripts/local_http.ps1

```diff
@@ -36,12 +56,33 @@ $uri = $uriBuilder.ToString()
 
 try {
     $httpMsg = New-Object System.Net.Http.HttpRequestMessage -ArgumentList $httpMethod, $uri
-    if ($reqObj.Body) {
+    if ($reqObj._body) {
+        $requestContent = New-Object System.Net.Http.StringContent ([string] $reqObj._body)
         $httpMsg.Content = $requestContent
-        $requestContent = New-Object System.Net.Http.StringContent ([string] $reqObj.Body)
     }
     foreach ($prop in $reqObj.headers | Get-Member -MemberType NoteProperty | Select-Object -ExpandProperty Name) {
-        $httpMsg.Headers.Add($prop, $reqObj.headers.$prop)
+        $headerValue = $reqObj.headers.$prop
+        if (!$httpMsg.Headers.TryAddWithoutValidation($prop, $headerValue)) {
+            $headerFixed = $false
+            if ($httpMsg.Content) {
+                if ($prop -like "content-type") {
+                    $httpMsg.Content.Headers.ContentType = New-Object System.Net.Http.Headers.MediaTypeHeaderValue $headerValue
+                    $headerFixed = $true
+                } elseif ($prop -like "content-range") {

```

Comment:
Would you add some comment why content-range is handled here?
Usually content-range is for response headers, not for request headers.

See https://stackoverflow.com/questions/716680/difference-between-content-range-and-range-headers more information.